.github/workflows/maven.yml

```diff
@@ -20,5 +20,7 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 1.8
+    - name: settings.xml
+      run: cat /home/runner/.m2/settings.xml
     - name: Build with Maven
-      run: mvn clean install --batch-mode  -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn --file pom.xml
+      run: mvn clean install -B -U -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn --file pom.xml
```

Comment:
clean is not needed, -U is also not needed.
"You can specify the runner type for each job in a workflow. Each job in a workflow executes in a fresh instance of the virtual machine. All steps in the job execute in the same instance of the virtual machine, allowing the actions in that job to share information using the filesystem."
from https://docs.github.com/en/actions/reference/specifications-for-github-hosted-runners

Reply:
done in develop